tests/network/secondary_network_dns/test_secondary_network_dns.py

```diff
@@ -57,11 +59,12 @@ def secondary_network_in_nslookup_output(
     # wildcard.apps.{cluster_base_domain} is the API access from outside the cluster
     sampler = TimeoutSampler(
         wait_timeout=TIMEOUT_40SEC,
-        sleep=TIMEOUT_5SEC,
+        sleep=TIMEOUT_2SEC,
```

Comment:
We decided for some time now to stop using these useless constants. Now you even add a new one and go beyond our sig control.

Just use something close, like 1sec as a 3x3sec is already embedded in the command.

We should get rid of these in followup code cleanups.